src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -398,9 +398,9 @@ def toggle_sliced_text(text, slice_length = 50, id = "toggle_sliced_text_#{Time.
     short + long
   end
 
-  def tab_link(label, path, active = false)
+  def tab_link(label, path, active = false, permit = true)
```

Comment:
The issue is that interconnect is now its own controller, but it's still under the `configuration` path. I am starting to wonder why do we have a tab for `Configuration` and another for `Interconnect`. I suggest we move the `Interconnect` form under `Configuration`. What do you think?

Reply:
I'm not sure about that, but if I remove the configuration scope from the interconnect?, because the others "tabs" are also outside of configuration

Reply:
I prefer to leave it under configuration. It makes sense for the user independent where the code is.

Reply:
Fine, no worries.